src/nanosvg.h

```diff
@@ -29,6 +29,33 @@
 #ifndef NANOSVG_H
 #define NANOSVG_H
 
+#include <math.h>
```

Comment:
Can you let me know which one? When you include the header without `NANOSVG_IMPLEMENTATION` there's no external dependencies needed. That define should be defined before including the header in one cpp file, just once. And then `math.h` get's included.